api/cmd/transformer/.env.sample

```diff
@@ -2,12 +2,13 @@ LOG_LEVEL=ERROR
 
 STANDARD_TRANSFORMER_CONFIG={"transformerConfig":{"preprocess":{"inputs":[{"variables":[{"name":"customer_id","jsonPath":"$$.customer_id"},{"name":"customer_name","jsonPath":"$$.customer_name"}]},{"tables":[{"name":"driver_table","baseTable":{"fromJson":{"jsonPath":"$$.drivers[*]","addRowNumber":true}},"columns":[]}]},{"feast":[{"tableName":"driver_feature_table","project":"merlin","entities":[{"name":"merlin_test_driver_id","valueType":"STRING","jsonPath":"$$.drivers[*].id"}],"features":[{"name":"merlin_test_driver_features:test_int32","valueType":"INT32","defaultValue":"-1"},{"name":"merlin_test_driver_features:test_float","valueType":"FLOAT","defaultValue":"0"}]}]}],"transformations":[{"tableTransformation":{"inputTable":"driver_table","outputTable":"driver_table","steps":[{"sort":[{"column":"row_number","order":"DESC"}]},{"renameColumns":{"row_number":"rank","id":"merlin_test_driver_id"}},{"updateColumns":[{"column":"customer_id","expression":"customer_id"}]},{"selectColumns":["customer_id","merlin_test_driver_id","rank"]}]}},{"tableJoin":{"leftTable":"driver_table","rightTable":"driver_feature_table","outputTable":"result_table","how":"LEFT","onColumn":"merlin_test_driver_id"}},{"tableTransformation":{"inputTable":"result_table","outputTable":"result_table","steps":[{"sort":[{"column":"rank","order":"ASC"}]},{"renameColumns":{"merlin_test_driver_id":"driver_id"}},{"selectColumns":["rank","driver_id","customer_id","merlin_test_driver_features:test_int32","merlin_test_driver_features:test_float"]}]}}],"outputs":[{"jsonOutput":{"jsonTemplate":{"fields":[{"fieldName":"instances","fromTable":{"tableName":"result_table","format":"SPLIT"},"fields":[]}]}}}]},"postprocess":{"outputs":[{"jsonOutput":{"jsonTemplate":{"fields":[{"fieldName":"instances","fromJson":{"jsonPath":"$$.model_response"},"fields":[]},{"fieldName":"instances_data","fromJson":{"jsonPath":"$$.model_response"},"fields":[]},{"fieldName":"timestamp","expression":"Now()","fields":[]}]}}}]}}}
 
-MERLIN_TRANSFORMER_MODEL_NAME=echo-2
-MERLIN_TRANSFORMER_MODEL_PREDICT_URL=localhost:8081
-MERLIN_TRANSFORMER_PORT=8080
+MERLIN_TRANSFORMER_MODEL_NAME=echo
+MERLIN_TRANSFORMER_MODEL_PREDICT_URL=localhost:8082
+MERLIN_TRANSFORMER_PORT=8081
 MODEL_HYSTRIX_RETRY_COUNT=0
 
-FEAST_SERVING_URL=localhost:6566
+DEFAULT_FEAST_SERVING_ENDPOINT=localhost:6566
+FEAST_SERVING_ENDPOINTS=localhost:6566,localhost:6567
```

Comment:
is this still being used?

Reply:
It is. I've updated the environment name to match the code.